blockchain/tx_pool_test.go

```diff
@@ -341,19 +370,12 @@ func TestInvalidTransactions(t *testing.T) {
 	pool.gasPrice = big.NewInt(1000)
 
 	// NOTE-Klaytn We only accept txs with an expected gas price only
-	//         regardless of local or remote.
-	// TODO : Need to KIP-71 hardfork.
-	//if err := pool.AddRemote(tx); err != ErrInvalidUnitPrice {
-	//	t.Error("expected", ErrInvalidUnitPrice, "got", err)
-	//}
-	//if err := pool.AddLocal(tx); err != ErrInvalidUnitPrice {
-	//	t.Error("expected", ErrInvalidUnitPrice, "got", err)
-	//}
-	if err := pool.AddRemote(tx); err != ErrGasPriceBelowBaseFee {
-		t.Error("expected", ErrGasPriceBelowBaseFee, "got", err)
+	// regardless of local or remote.
```

Comment:
Don't we need to have test cases for both before KIP-71 and after KIP-71?

Reply:
I agree. It is added in `TestInvalidTransactionsKip71`